spec/controllers/ministers_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe MinistersController, type: :controller do
+  describe 'GET #index' do
+    let!(:health) { Fabricate(:minister, name: 'Minister for Health') }
+    let!(:pm) { Fabricate(:minister, name: 'Prime Minister') }
+    let!(:comms) { Fabricate(:minister, name: 'Minister for Communications') }
+
+    before { get :index }
+
+    it { expect(subject.ministers).to eq [pm, comms, health] }
```

Comment:
You should probably test that the controller render was successful.

    it { is_expected.to be_success }